site/static/main.ts

```diff
@@ -29,8 +29,10 @@ function renderExample($target: Selection<any, any, any, any>, text: string) {
 
   const vis = $target.append('div').attr('class', 'example-vis');
 
+  // Decrease visual noise by removing $schema and description from code examples.
+  const textClean = text.replace(/(\s)+\"(\$schema|description)\": \".*?\",/g, '');
```

Comment:
I assume the `$schema` is still included when users click "Open in Vega Editor"?  If so, I think this is a good idea :)